source/adminguide/templates/_cloud_init.rst

```diff
@@ -75,15 +65,15 @@ These features can be implemented in `“Linux template creation process” <_cr
 	   
 #. **Password management**
 
-   Cloudstack integration with cloud-init `set-passwords module <https://cloudinit.readthedocs.io/en/latest/topics/modules.html?highlight=ssh_pwauth#set-passwords>`_ will enable the platform to set a password for each instance created from the Main Template. Additionally it will allow to reset the user password through the GUI.
+   Cloudstack integration with cloud-init `set_passwords module <https://cloudinit.readthedocs.io/en/latest/topics/modules.html?highlight=ssh_pwauth#set-passwords>`_ will enable the platform to set a password for each instance created from the Main Template. Additionally it will allow to reset the user password through the GUI.
    
-   - **Enable set-passwords module on every boot**
+   - **Enable set_passwords module on every boot**
    
-     By default the set-passwords module runs only on first boot of the instance, change that to run on every boot.
+     By default the set_passwords module runs only on first boot of the instance, change that to run on every boot.
    
      .. code:: bash
    
-      sudo sed -i s/" - set-passwords"/" - [set-passwords, always]"/g /etc/cloud/cloud.cfg
```

Comment:
@TheRealFalcon 
in my template (the cloud-init version is 23.1.2), it still uses `set-passwords` 
would be good to mention the cloud-init version of old/new setting ?

Reply:
looks good